FWCore/Framework/src/PrincipalCache.cc

```diff
@@ -157,6 +157,13 @@ namespace edm {
     }
   }
 
+  void
+  PrincipalCache::readFile() {
```

Comment:
Is this really causing the file to be read? If it is only something that should happen after the file has been read, I'd prefer a different function name.

Reply:
Done. I changed the function name to preReadFile.